docs/release_notes/feb2023.md

```diff
@@ -8,6 +8,67 @@ tags:
   - changelog
 ---
 
+## 2.8.0.0 (February 28, 2023)
+
+- Prophecy Python libs version: 1.4.2
+- Prophecy Scala libs version: 7.0.7
+
+### Updates {#UpdatesRelease280}
+
+#### No team required on sign up
+
+Previously, A user was asked to enter a unique team when signing up for Prophecy. To improve the sign-up experience, requirement of providing a Team is now removed.  
+Now the user can just sign up by giving his Name, Email and Company name. He can be added to a team by Admin post sign up.
+
+![New_Sign_up](img/new_sign_up.png)
+
+#### Sorting on tables
+
+All metadata tables will now have options to sort on the columns. By default, the entities in table will be sorted on Modified at or Created At time in decreasing order.
+
+![Sorting](img/Sorting.gif)
+
+#### Use a Cluster in Current tab
+
+A User editing Prophecy Pipelines in multiple tabs, can now detach the cluster Attached to a Pipeline in another tab, and attach to current Pipeline by clicking the same cluster.
```

Comment:
"tabs, can" -> "tabs can"
"tab, and" -> "tab and"
"attach to current Pipeline" -> "attach the current Pipeline"